### DISCUSSION TITLE
Why can't I get globalData?

### DISCUSSION THREAD

**Discussion by kumuweifengchun-sudo** [kaliiiiiiiiii/Selenium-Driverless#299] [Q&A]
![image](https://github.com/user-attachments/assets/5c1ddbaf-7c2e-425d-b370-f305cf80f260)
When I execute a script like this, the value of globalData is fetched normally in selenium, but the value returned in the library is indeed None

**Comment by ttlns** [comment]
See https://github.com/kaliiiiiiiiii/Selenium-Driverless?tab=readme-ov-file#isolated-execution-contexts
